packages/transport-webtransport/src/index.ts

```diff
@@ -93,15 +110,32 @@ class WebTransportTransport implements Transport {
             return
           }
 
-          wt.close(closeInfo)
+          this.metrics?.dialerEvents.increment({ close: true })
+          wt.close()
```

Comment:
Whyd we get rid of closeInfo

Reply:
We use it as a cheap error object, but that's not what it does and it affects the WebTransport after closing.

From step 9 in the [cleanup procedure](https://www.w3.org/TR/webtransport/#webtransport-procedures):

> If closeInfo is given, then set transport.[[[State]]](https://www.w3.org/TR/webtransport/#dom-webtransport-state-slot) to "closed". Otherwise, set transport.[[[State]]](https://www.w3.org/TR/webtransport/#dom-webtransport-state-slot) to "failed".

Reply:
ok makes sense, good to know